src/packets_input.c

```diff
@@ -98,6 +98,13 @@ TbBool process_dungeon_control_packet_dungeon_build_room(long plyr_idx)
     }
     get_dungeon_build_user_roomspace(&playeradd->render_roomspace, player->id_number, player->chosen_room_kind, stl_x, stl_y, playeradd->roomspace_mode);
     long i = tag_cursor_blocks_place_room(player->id_number, stl_x, stl_y, player->full_slab_cursor);
+    if ( (playeradd->roomspace_mode == drag_placement_mode) && (player->chosen_room_kind != RoK_BRIDGE) )
```

Comment:
please avoid using hardcoded roomtypes in new code, and do everything using roles
I'm working on a ticket to do the same for existing code
(player->chosen_room_kind != RoK_BRIDGE) would be replaced with (!room_role_matches(player->chosen_room_kind,(RoRoF_PassWater | RoRoF_PassLava)))